### PR TITLE
Add demo server and some demos

### PR DESCRIPTION
**Pull request by keirog**
@lc512k @GlynnPhillips

### CODE REVIEW

File: demos/demo.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+			<h2>Light</h2>
+			{{> n-teaser/templates/light}}
+		</div>
+		<div data-o-grid-colspan="12 M4">
+			<h2>Standard</h2>
+			{{> n-teaser/templates/standard}}
+		</div>
+		<div data-o-grid-colspan="12 M4">
+			<h2>Heavy</h2>
+			{{> n-teaser/templates/heavy colspan='{"default": 12, "L": 6}' position='{"default": "bottom"}' widths="[500, 332]" lazyLoad=false}}
```
Comment (andygnewman) [inline demos/demo.html]: the colspan and position values are not required for the images - they were n-card specific

Comment (keirog) [inline demos/demo.html]: oh, ta

**Comment by keirog** [comment]
(cc @andygnewman for the last commit)

**Comment by andygnewman** [comment]
Looks good to me. 👍

**Pull request merged**